tools/codegen/selective_build/operator.py

```diff
@@ -14,3 +14,22 @@ class SelectiveBuildOperator():
     # use the information to generate code in VariableType_N.cpp for registration
     # of training related operators
     is_used_for_training: bool
+
+    @staticmethod
+    def combine(
```

Comment:
Couldn't this just be a method? :)

Reply:
Fair point! I didn't know what the preference was - i.e. it's not an instance method since it doesn't mutate anything and operates just on arguments - and since it's a pure functional method, it wasn't clear to me if it should be a static method or just a free-standing function. I am guessing the preference is to make it the latter.

Reply:
We have no particular injunction against purely functional methods :) And since most of our dataclasses are frozen, they couldn't be mutable anyway.